api/pkg/apps/iam/static.go

```diff
@@ -677,6 +717,24 @@ var SecondaryPhoneNumberAttribute = Attribute{
 	},
 }
 
+var TertiaryPhoneNumberAttribute = Attribute{
+	ID:         "4a0ba072-66a5-403f-bea1-35e9427659fb",
+	Name:       "tertiaryPhoneNumber",
+	Type:       form.Phone,
+	Attributes: form.FormElementAttributes{Name: "tertiaryPhoneNumber"},
+	Translations: []AttributeTranslation{
+		{
+			Locale:           "en",
+			LongFormulation:  "Tertiary phone number",
+			ShortFormulation: "Tertiary phone number",
+		},
+	},
+	IsPersonallyIdentifiableInfo: false,
```

Comment:
This should be marked as PII, according to GDPR.

![PII-Personal-Data-1- 001-1024x576](https://user-images.githubusercontent.com/15915453/131687400-160da0b8-f77b-434a-be99-dd021e105134.jpeg)
[source](https://techgdpr.com/blog/difference-between-pii-and-personal-data/)